EmpireLegends/Player.cpp

```diff
@@ -234,3 +235,11 @@ Resources* Player::getResources() const
 	return pResources;
 }
 
+void Player::addGoods(Good* addedGood)
+{
+	addedGood->applyGood(pResources);
+	
+	//Just replace this line with a call to the applyGood method with the Resources instance	
+	//pResources->applyGood(addedGood);
+
```

Comment:
maybe just remove the comments. also, now that it's modified, I feel like `addGoods` should be renamed to something else, like `applyGood`